src/ui/components/tables/CCStressTable.vue

```diff
@@ -182,7 +182,6 @@ export default class CCStressTable extends Vue {
   dialog = false
   show(): void {
     this.dialog = true
-    if (this.mech.CurrentStress === 0) this.window = 4
```

Comment:
This line was obviously meant to enforce reactor criticality with 0 stress remaining, but was prevented from working by the return upon 0 stress here: https://github.com/massif-press/compcon/blob/ecfbafba246f73ae1def076cd23ee677b3a2f076/src/features/pilot_management/ActiveSheet/layout/MechBlock.vue#L611